Tests/InstallationTests/ReceiptParserInstallation/ReceiptParserIntegration/ReceiptParserIntegrationApp.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ReceiptParserIntegrationApp.swift
+//  ReceiptParserIntegration
+//
+//  Created by Nacho Soto on 11/29/22.
+//
+
+import SwiftUI
+
+import ReceiptParser
+
+@main
+struct ReceiptParserIntegrationApp: App {
+
+    init() {
+        let parser = PurchasesReceiptParser.default
+        let _: AppleReceipt? = try? parser.parse(from: Data())
+    }
+
+    var body: some Scene {
+        WindowGroup {
+            EmptyView()
```

Comment:
Hmm, do we need this file? I guess we need the project in order to have the tests, but not sure if the project requires an app class like this which doesn't seem useful right now.

Reply:
I made this project as slim as possible. The other integration test apps have even more stuff in them.